### DISCUSSION TITLE
Defined Camera Positions, Rotation of HDR, True Top View

### DISCUSSION THREAD

**Discussion by dnderk** [google/model-viewer#2651] [Q&A]
Hello,

I'm pretty new to all of that but after testing what I found in the Documentation couldn't find solutions or get it working. 
Is it possible to either pre-set camera positions (eg. Front, Right, Top) or add a field where users can set a custom camera position?
Is it possible to add a slider that allows users to rotate the HDR?
Is it possible to extend the vertical flexibility of the camera? Trying to get a straight top view doesn't work for me. 

Thanks,

**Comment by elalish** [comment]
For 1) https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-cameraOrbit
For 2) https://modelviewer.dev/examples/stagingandcameras/#turnSkybox
For 3) https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-maxCameraOrbit
